sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/TestEnvironment.cs

```diff
@@ -16,6 +16,23 @@ namespace Azure.Messaging.EventHubs.Tests
     ///
     public static class TestEnvironment
     {
+        /// <summary>The name of the shared access key to be used for accessing an Event Hubs namespace.</summary>
+        public const string EventHubsDefaultSharedAccessKey = "RootManageSharedAccessKey";
+
+        /// <summary>The default value for the maximum duration, in minutes, that a single test is permitted to run before it is considered at-risk for being hung.</summary>
+        private const int DefaultPerTestExecutionLimitMinutes = 5;
+
+        /// <summary>The environment variable value, or default, for the maximum duration, in minutes, that a single test is permitted to run before it is considered at-risk for being hung, lazily evaluated.</summary>
+        private static readonly Lazy<TimeSpan> EventHubsPerTestExecutionLimit =
+            new Lazy<TimeSpan>(() =>
+            {
+                var interval = DefaultPerTestExecutionLimitMinutes;
+                int.TryParse(Environment.GetEnvironmentVariable("EVENT_HUBS_PER_TEST_LIMIT_MINUTES"), out var environmentVariable);
```

Comment:
Where does this `out var environmentVariable` parameter get used? Or is it a throwaway `_`?

Reply:
Was the intent here to have the value of `environmentVariable` override the default if specified?

Reply:
derp.   Yes.   That's my bad.   The intent is for the environment variable to override the default for `interval`, and I apparently rebased that commit away rather than squashing.

Reply:
... and I've now added that back in.